.travis.yml

```diff
@@ -14,13 +17,11 @@ matrix:
   allow_failures:
     - rvm: rbx-3
     - rvm: ruby-head
-sudo: false
```

Comment:
This used to be necessary to make use of newer Travis infrastructure (iirc, without this, the test would run on a VM). Is that no longer the case?

Reply:
@bklang Yes, it looks like it's now deprecated. I was going off of this info from https://docs.travis-ci.com/user/reference/overview/#deprecated-virtualization-environments :

> If you’re trying to use sudo: false or dist: precise keys in your travis.yml, we recommend you switch to our current [Xenial Linux infrastructure](https://docs.travis-ci.com/user/reference/xenial/).

I've already seen in bef35c1 that the build works great on Xenial (apart from Rubinius, which was already broken).  Should we just go with Xenial?

Reply:
👍 for Xenial

Reply:
@bklang Thank you sir!  Ready for re-reviewski :monocle:

Side note: Why is there no monocle emoji? 😞